bentoml/spacy.py

```diff
@@ -391,24 +413,57 @@ def _setup(self) -> None:  # type: ignore[override]
             config=self._config,
         )
 
-    # pylint: disable=arguments-differ
-    def _run_batch(  # type: ignore[override]
+    @overload  # pragma: no cover
```

Comment:
this `@overload` change sacrifices readability too much, I'd just ignore type checking for it if it has to be done this way.